### DISCUSSION TITLE
希望mqAdmin支持通过proxy 连接进行性能测试

### DISCUSSION THREAD

**Discussion by charlienss** [apache/rocketmq#6638] [Q&A]
希望mqAdmin支持通过proxy 连接进行性能测试

**Comment by drpmma** [comment]
Related discussion #6603

**Comment by drpmma** [comment]
There's no plan to support admin interface in proxy for now. Please use nameserver as access endpoint instead.

**Comment by drpmma** [comment]
close for duplication #6620
